lib/layout/adaptive.dart

```diff
@@ -9,9 +9,7 @@ import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
 /// build adaptive and responsive layouts.
 bool isDisplayDesktop(BuildContext context) {
```

Comment:
You could make this even shorter
```suggestion
bool isDisplayDesktop(BuildContext context) => getWindowType(context) >= AdaptiveWindowType.m;
```

Reply:
Done